view/frontend/templates/js/connect.phtml

```diff
@@ -24,24 +24,14 @@ if ($block->shouldDisableBoltCheckout()) return;
 //check if we need Bolt on this page
 if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled() && !$block->isBoltProductPage()) return;
 
-$trackJsUrl   = $block->getTrackJsUrl();
 $connectJsUrl = $block->getConnectJsUrl();
 $checkoutKey  = $block->getCheckoutKey();
 ?>
-<script>console.log("Bolt M2 Version: <?=$this->getModuleVersion()?>");</script>
-<script
-    id="bolt-track"
-    type="text/javascript"
-    src="<?php echo $trackJsUrl; ?>"
-    data-shopping-cart-id="magento2"
-    async
-    data-publishable-key="<?php echo $checkoutKey; ?>">
-</script>
 <script
     id="bolt-connect"
     type="text/javascript"
     src="<?php echo $connectJsUrl; ?>"
     data-shopping-cart-id="magento2"
-    async
+    defer

```

Comment:
@daisy1754 This is to postpone the execution of connect.js to after the page is fully parsed (having our javascript in place), preventing button rendering too early, thus lowering the chances of having dead clicks. Especially on slow networks. Even this way there is no noticeable delay in showing the button - it's available way before every (async) parts of the page load. It's a micro optimization I added after acknowledged how seriously dead clicks are treated. However, this may not be needed since the connect.js is now moved to the end of body. Anyway, it does no harm I think.